test/Lazy/intersectionBy.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { toArray, toAsync } from "../../src";
+import { Concurrent } from "../../src/Lazy/concurrent";
+import intersectionBy from "../../src/Lazy/intersectionBy";
```

Comment:
```suggestion
import { intersectionBy, toArray, toAsync } from "../../src";
import { Concurrent } from "../../src/Lazy/concurrent";
```
This code can test whether `intersectionBy` has been exported from the `index` well.

Reply:
good 👍